src/detokenization.rs

```diff
@@ -0,0 +1,50 @@
+use crate::Task;
+use serde::{Deserialize, Serialize};
+
+/// Input for a [crate::Client::detokenize] request.
+pub struct TaskDetokenization<'a> {
+    /// List of token ids which should be detokenized into text.
+    pub token_ids: &'a Vec<u32>,
+}
+
+#[derive(Serialize, Debug)]
+struct DetokenizationRequest<'a> {
```

Comment:
Please rename this to `DetokenizationBody`.

Reply:
OK, I renamed it to `BodyDetokenization` and `ResponseDetokenization` in the spirit of the existing `BodyCompletion` and `ResponseCompletion`.